src/diracx/core/models.py

```diff
@@ -67,6 +67,11 @@ class JobStatus(str, Enum):
     RESCHEDULED = "Rescheduled"
 
 
+class JobMinorStatus(str, Enum):
```

Comment:
Can be taken from https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/WorkloadManagementSystem/Client/JobMinorStatus.py